Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -82,7 +82,7 @@ namespace Terrain
 
         if (auto serialize = azrtti_cast<AZ::SerializeContext*>(context))
         {
-            serialize->Class<TerrainPhysicsColliderConfig, AZ::ComponentConfig>()
+            serialize->Class<TerrainPhysicsColliderConfig>()
```

Comment:
question: Does this not require a version bump and a converter step to fixup the old data? Or are things at a stage where that isn't required yet (very early)

Reply:
There's no conversion for game time components since all prefabs should reprocess due to the fingerprint change of the Editor component.